.github/2.0/cookbooks/CleanCode.md

```diff
@@ -1,7 +1,7 @@
 # Cookbook on Clean Code
```

Comment:
Shouldn't we follow the same guidelines as for other READMES? So title case here too?

Reply:
Title case doesn't mean every word capitalized :) This is in title case already

[Minor words shouldn't be capitalized](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)